Sources/API/YouTubePlayer+EventAPI.swift

```diff
@@ -6,7 +6,7 @@ import Foundation
 public extension YouTubePlayer {
     
     /// The current YouTubePlayer State, if available
-    var state: State? {
+    @_spi(Internals) public var state: State? {
```

Comment:
`@spi(Internals)` can be omitted since the `var state: State?` is declared in a `public` extension of the `YouTubePlayer`

Reply:
:O you're right! I feel embarrassed that commit 48ea5df1f10886c7ea53938b1d840f5e45b07459 was ever made public